src/shared/components/ResourceEditor/index.tsx

```diff
@@ -199,6 +175,26 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
     setEditing(false);
   };
 
+  useEditorTooltip({
+    orgLabel,
+    projectLabel,
+    isEditing,
+    ref: codeMirorRef,
+  });
+  useEditorPopover({
+    orgLabel,
+    projectLabel,
+    ref: codeMirorRef,
+  });
+
+  React.useEffect(() => {
+    return () => {
+      if (location.pathname !== DATA_EXPLORER_GRAPH_FLOW_PATH) {
```

Comment:
Hmm, I like the idea of clearing the cache, but I'm not sure that we should clear the cache everythime the user navigates away from `data-explorer/graph-flow`. I think we should clear the cache only if the user navigates away from `data-explorer` (i.e. `!location.pathname.includes('data-explorer')`). Wdyt?


*Reason:*


Based on the user tests, I have a feeling that the user will often click on row in data-explorer, check out the graph-flow, come back to data-explorer table, then click another or same row again. There's a high chance that a lot of same links would need to be resolved but since the cache is cleared, we won't benefit from it.